### DISCUSSION TITLE
EPG For Tivimate

### DISCUSSION THREAD

**Discussion by driftflow23** [iptv-org/community#580] [Q&A] [answered]
Added this to tivimate and will like to add some channels from this project to my favorites. But just wanted to know if there was an epg that could be linked so I can use for the channels cause they all say no information.

**Comment by BellezaEmporium** [comment, marked as answer]
We do have an EPG repository available in https://github.com/iptv-org/epg, however you will need to create the guides yourself. Everything you need to know is in the README.

**Comment by Melulup** [comment]
Hello, how are you? Congratulations on this very entertaining channel. I wanted to mention that I have followed the tutorial from "https://github.com/iptv-org/epg" and when I get to the step to download the guide (npm run grab -- --site=example.com), I don't know which guide to choose instead of example.com. I have downloaded the playlist of all the countries in the world, classified by country. So, if I want the guide for all the countries, do I have to do the step one by one to download all the guides?

**Reply by BellezaEmporium** [reply]
Hello, yes that's the goal.

Usually people tend to automate that to be sure to get them all.
